lib/ansible/cli/doc.py

```diff
@@ -1144,6 +1144,11 @@ def get_role_man_text(self, role, role_json):
                 text.append("%s\n" % textwrap.fill(DocCLI.tty_ify(desc),
                                                    limit, initial_indent=opt_indent,
                                                    subsequent_indent=opt_indent))
+            if doc.get('version_added'):
+                version_added = doc.pop('version_added')
+                version_added_collection = doc.pop('version_added_collection', None)
+                text.append("ADDED IN: %s\n" % DocCLI._format_version_added(version_added, version_added_collection))
```

Comment:
This is adding support for a new field so it needs a minor_changes changelog: https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment.

Reply:
It is best to move this part to a separate PR. The docs PR can (and should) be backported, but this part cannot be backported as its a new feature.

Reply:
@felixfontein we're past the point where I'm backporting docs fixes to stable-2.14. As such, can this PR go ahead (once a changelog is added ) so it's part of stable-2.15?

Reply:
@samccann this is kind of a new feature so it cannot be backported anyway. It should really be moved to a separate PR, to separate it from the docs part of this PR which is essentially unrelated to this change.

Reply:
This has now been moved to #80625, as suggested.